spec/game_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'game'
+
+describe Game do
```

Comment:
Does this need more testing to simulate losing and drawing?

Reply:
Agreed :)